lib/src/main/java/graphql/nadel/engine/transform/hydration/batch/NadelNewBatchHydrator.kt

```diff
@@ -497,6 +528,26 @@ internal class NadelNewBatchHydrator(
         }
     }
 
+    private fun getFilteredInstructions(
+        instructions: List<NadelBatchHydrationFieldInstruction>,
+        // fieldSource: ValueSource.FieldResultValue,
+        sourceObject: JsonNode,
+        state: State,
```

Comment:
Please remove `state` and add `context(NadelBatchHydratorContext)` to the function instead.

When you do that you can access `aliasHelper` directly.

Reply:
I see, thanks